Dockerfile

```diff
@@ -80,6 +80,7 @@ ADD https://dl.google.com/android/repository/sdk-tools-linux-4333796.zip /builds
 ADD https://github.com/taskcluster/generic-worker/releases/download/v16.0.0/generic-worker-simple-linux-amd64 /usr/local/bin/generic-worker
 ADD https://github.com/taskcluster/livelog/releases/download/v1.1.0/livelog-linux-amd64 /usr/local/bin/livelog
 ADD https://github.com/taskcluster/taskcluster-proxy/releases/download/v5.1.0/taskcluster-proxy-linux-amd64 /usr/local/bin/taskcluster-proxy
+ADD https://github.com/taskcluster/taskcluster-worker-runner/releases/download/v1.0.2/start-worker-linux-amd64 /usr/local/bin/start-worker
```

Comment:
12 days old with v1.0.4 available. Should we stay on the bleeding edge?

Reply:
Yes please - v1.0.4 adds a security check.

Reply:
Updated and retested.